.github/workflows/python-package.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: debian-latest
     strategy:
       matrix:
         python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
I'm using f-strings and I think that's only available on python > 3.6

Reply:
ok, that will fail once we have tests for taht code